pkg/test/helpers.go

```diff
@@ -197,6 +198,21 @@ func BuildTLSSecret(vdb *vapi.VerticaDB, name, key, cert, rootca string) *corev1
 	return secret
 }
 
+func CreateFakePemTLSSecret(ctx context.Context, vdb *vapi.VerticaDB, c client.Client, name string, caCert, cert security.Certificate) {
```

Comment:
Lets use `CreateFakeTLSSecret` instead.

Reply:
It complains `failed to show restore points: fail to show restore points: execute NMAHealthOp failed, details: fail to load HTTPS certificates, details tls: failed to find any PEM data in certificate input` if I use CreateFakeTLSSecret "https://github.com/vertica/vertica-kubernetes/actions/runs/7752956063/job/21143979568"

Reply:
As discussed in the scrum meeting, we should create a 'dummy' function to avoid calling vclusterOps for the unit tests.